src/routes/areas/createArea.ts

```diff
@@ -38,33 +40,24 @@ export const handler: RouteHandler<CreateAreaRoute> = async (req, reply) => {
     reply.status(200).send();
   } catch (error) {
     return reply.status(500).send({
-      error: {
-        code: 'unknown',
-        message: `An unknown error occurred when trying to create an Area. Error: ${error}`
-      }
+      error: `${error}`,
```

Comment:
ummm I thought we were returning an error with this structure:

```js
{
error: ErrorSchema
}
```

am I wrong?  🤔